recipes/toolchain/activate.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    # for Mac OSX
+    export CC=clang
+    export CXX=clang++
+    export MACOSX_VERSION_MIN="10.7"
+    export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+    export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LDFLAGS="${LDFLAGS} -stdlib=libc++ -lc++"
+    export LINKFLAGS="${LDFLAGS}"
+elif [ "$(uname)" == "Linux" ]
+then
+    # for Linux
+    export CC=gcc
+    export CXX=g++
+    export CXXFLAGS="${CXXFLAGS}"
```

Comment:
There is no PR yet because I am trying to get the recipe to build locally first, and in the process I noticed that there were some other bugs in the project that needed fixing.  I put in a PR to cyclus/cyclus#1253, but I am waiting for that to be merged.